sql/2022/sustainability/stylesheet_count.sql

```diff
@@ -13,16 +13,18 @@ try {
 
 SELECT
   _TABLE_SUFFIX AS client,
+  percentile,
   COUNTIF(stylesheets.remote = 1) / COUNT(0) AS pct_1_remote,
-  SUM(stylesheets.inline) AS num_inline_stylesheets,
-  SUM(stylesheets.remote) AS num_inline_stylesheets
+  APPROX_QUANTILES(stylesheets.inline, 1000)[OFFSET(percentile * 10)] AS num_inline_stylesheets,
+  APPROX_QUANTILES(stylesheets.remote, 1000)[OFFSET(percentile * 10)] AS num_inline_stylesheets
 FROM (
   SELECT
     _TABLE_SUFFIX,
     url,
     getStylesheets(payload) AS stylesheets
   FROM
     `httparchive.pages.2022_06_01_*`
+    UNNEST([10, 25, 50, 75, 90, 100]) AS percentile
 )
```

Comment:
@tunetheweb can you please confirm if this is on the right track for updating to use averages rather than percentiles? apologies I am still fairly new to SQL

Reply:
Pretty much. Though the SQL needs a little cleaning up. Resolving this comment and will make another.